# About me
Hello everyone!
I am working non-stop in wet-lab laboratories since 2009 where I started as a trained lab technician. I finished my PhD december 2022 in the group of [Prof. Rob Russell](http://russelllab.org). My job there included managing 'my own' laboratory, which includes supervising a permanent technical assistant & the occasional students.

# Research
My research focus lies on interrogating human health and disease & I try to use my insights and knowledge to answer questions related to this using wet-lab and dry-lab approaches. 

During my PhD I started to delve into **Python** and **R** and I try to moderate between the wet-lab point of view and the bioinformation point of view when approaching bioinformatics problems.

# Current Projects 
## Kinase Resistance
Collaborative project with other members of the Russell lab.
Will be published & made online soon. 

More info will follow once ``Kinase Resistance`` is published and the repositories go public.


## Annotating Alignments (standalone)
A variant of this is already used in the ``Kinase Resistance`` project. The version here will be more generalized and hence hopefully more useful for the wider scientific community. 

More info will follow once ``Kinase Resistance`` is published and the repositories go public.

## PROTEORIZER
This is my pet project. It follows a holistic approach by combining a trove of publicly available information on a protein of interest to determine whether a variants may be functional, and then also providing hints at how the particular variants may influence protein function. I am using **data mining** as well as **interrogation of 3D structures** combined with **naive bayesian combination** and a **random forest classifier** to arrive at a verdict. This aproach has a **fully functional web app** where the user simply provides a protein (gene name or uniprot accession) and a variant (or multiple variants), coded in **R-Shiny**.

More info will follow once PROTEORIZER is published and the repository goes public.

# Paper Selection

- ``Schmenger, T., Diwan, G.D., Singh, G. et al. Never-homozygous genetic variants in healthy populations are potential recessive disease candidates. npj Genom. Med. 7, 54 (2022). https://doi.org/10.1038/s41525-022-00322-z``

- ``Samantha Ebersoll, Blessing Musunda, Torsten Schmenger, Natalie Dirdjaja, Mariana Bonilla, Bruno Manta, Kathrin Ulrich, Marcelo A. Comini, R. Luise Krauth-Siegel,
A glutaredoxin in the mitochondrial intermembrane space has stage-specific functions in the thermo-tolerance and proliferation of African trypanosomes, Redox Biology, Volume 15, 2018, https://doi.org/10.1016/j.redox.2018.01.011``

- ``Osswald, M., Jung, E., Sahm, F. et al. Brain tumour cells interconnect to a functional and resistant network. Nature 528, 93â€“98 (2015). https://doi.org/10.1038/nature16071``




