# About me
Hello everyone!
I was working non-stop in wet-lab laboratories between 2009 and 2024. In 2009 I started my training to become a lab technician and later I finished my PhD in december 2022 in the group of [Prof. Rob Russell](http://russelllab.org). My job there included managing 'my own' laboratory, which included supervising a permanent technical assistant & the occasional students. I am now a project engineer in nuclear waste storage and disposal, and experiment with Javascript and VBA.

<!-- ![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=tschmenger&layout=compact) -->

# Research
My research focus still lies on interrogating human health and disease.

During my PhD I started to delve into **Python** and **R** and I am experienced in moderating between the wet-lab and the bioinformation point of view, when approaching big data problems. 

# Latest Projects 
## Kinase Resistance
Collaborative project with other members of the Russell lab.
This is a data-driven approach to predict the functional consequence of genetic changes in protein kinases. You can find the [paper here](https://genomemedicine.biomedcentral.com/articles/10.1186/s13073-025-01564-z#citeas) and the associated webserver is called [Activark](http://activark.russelllab.org/). My contributions were the wet lab experiments, the alignment viewer (also see my other associated project below) and conceptual ideas (see [Proteorizer](https://shiny.russelllab.org/proteorizer/) for very familiar ideas).


## [Annotating Alignments (standalone)](https://github.com/tschmenger/Annotate_Alignments)
A variant of this is already used in the ``Kinase Resistance`` project. The version here will be more generalized and hence hopefully more useful for the wider scientific community. 
You can try my webserver [CALVI](https://schafft-wissen.org/CALVI/) or visit the repo to use a command line tool (legacy version).

This work does not yet have it's own technical paper, if you are using it please cite my proteorizer preprint or even better the kinase paper from above.

`Torsten Schmenger, Gaurav Diwan, Robert Bruce Russell. "PROTEORIZER: A holistic approach to untangle functional consequences of variants of unknown significance", https://doi.org/10.1101/2024.07.16.603688.`

## [PROTEORIZER](https://github.com/tschmenger/PROTEORIZER)
This is my pet project. It follows a holistic approach by combining a trove of publicly available information on a protein of interest to determine whether a variants may be functional, and then also providing hints at how the particular variants may influence protein function. I am using **data mining** as well as **interrogation of 3D structures** combined with **naive bayesian combination** and a **random forest classifier** to arrive at a verdict. This aproach has a **fully functional web app** where the user simply provides a protein (gene name or uniprot accession) and a variant (or multiple variants), coded in **R-Shiny**.

Please find my preprint here:

`Torsten Schmenger, Gaurav Diwan, Robert Bruce Russell. "PROTEORIZER: A holistic approach to untangle functional consequences of variants of unknown significance", https://doi.org/10.1101/2024.07.16.603688.`

This work heavily influenced [Activark](https://activark.russelllab.org/) and, of course, [CALVI](https://schafft-wissen.org/CALVI/).

## Several non-scientific pet projects
I have also started to work on other pet projects! 
- DELPHI: Check out [DELPHI](https://schafft-wissen.org/delphi.html), where I randomize on different random tables that are commonly used in Dungeons & Dragons 5th Edition. No more thumbing through books to find the right table. This is a **work in progress**, more tables may be added over time.
- Immunity, the game: I worked on my own [sidescrolling game](https://schafft-wissen.org/immunity.html), with educational purpose - of course. In this game you are a white blood cell. You will encounter different types of pathogens and must try to 'eat' them all! I offer 3 different game modes, one of them will have some educational content. This content may increase and get more fleshed out over time, so **work in progress** as well.
- Crypts and Champions, project working name '[Dungeon Delvers](https://schafft-wissen.org/dungeon_delvers.html)': Perhaps my most ambitious pet project, a turn-based and auto-battling fantasy dungeon crawler with roguelike elements. Check it out! It is completely written in browser friendly JS (+HTML +CSS of course). 

# Scientific Paper Selection
- ``Singh, G., Schmenger, T., Gonzalez-Sanchez, J.C. et al. Discriminating activating, deactivating and resistance variants in protein kinases. Genome Med 17, 133 (2025). https://doi.org/10.1186/s13073-025-01564-z``

- ``Schmenger, T., Diwan, G.D., Singh, G. et al. Never-homozygous genetic variants in healthy populations are potential recessive disease candidates. npj Genom. Med. 7, 54 (2022). https://doi.org/10.1038/s41525-022-00322-z``

- ``Samantha Ebersoll, Blessing Musunda, Torsten Schmenger, Natalie Dirdjaja, Mariana Bonilla, Bruno Manta, Kathrin Ulrich, Marcelo A. Comini, R. Luise Krauth-Siegel,
A glutaredoxin in the mitochondrial intermembrane space has stage-specific functions in the thermo-tolerance and proliferation of African trypanosomes, Redox Biology, Volume 15, 2018, https://doi.org/10.1016/j.redox.2018.01.011``

- ``Osswald, M., Jung, E., Sahm, F. et al. Brain tumour cells interconnect to a functional and resistant network. Nature 528, 93â€“98 (2015). https://doi.org/10.1038/nature16071``

# Links

[Researchgate](https://www.researchgate.net/profile/Torsten-Schmenger)

[Google Scholar](https://scholar.google.com/citations?user=OvgMdSgAAAAJ&hl=en)

[Linkedin](https://www.linkedin.com/in/dr-torsten-schmenger-85b79021b/)

[schafft-wissen.org](https://schafft-wissen.org/)



